canonical-websites/partners.ubuntu.com
# Move YUI to use https

Converted YUI CDN loads to https://yui-s.yahooapis.com
It caused issues with find-a-partner.

I have deployed this on staging to easily QA.

## QA

- Head to https://partners.staging.ubuntu.com/find-a-partner
- See no console errors and the page works correctly
